ivy/functional/ivy/experimental/layers.py

```diff
@@ -2189,6 +2189,8 @@ def reduce_window(
     # ToDo: add support for window_dilation
     computation = _correct_ivy_callable(computation)
     op, dims, strides = operand, window_dimensions, window_strides
+    if isinstance(strides, int):
+        strides = tuple([strides] * len(dims))
     init_value = _cast_init(init_value, op.dtype)
```

Comment:
Actually the same should be done for all the arguments: `window_dimensions`, `window_strides`, `padding`, `base_dilation` and `window_dilation`, since they could all be a single integer. 
And instead of using `len(dims)` I would suggest using `len(op.shape)` in case `dims` is also an integer.